cpp/refactor_module/algorithm/refactor.cpp

```diff
@@ -1,9 +1,18 @@
 #include "refactor.hpp"
 
-#include <mg_utils.hpp>
 #include <unordered_set>
+
+#include <mg_utils.hpp>
 #include "mgp.hpp"
 
+namespace {
+void ThrowInvalidTypeException(const mgp::Value &value) {
+  std::ostringstream oss;
+  oss << value.Type();
+  throw mgp::ValueException("Unsupported type for this operation, received type: " + oss.str());
+}
```

Comment:
Can we isolate this in some function, this is a repeating pattern. And did you try to link it with `fmt`?

Reply:
Isn't the C++ API the only place where we could add this so its usable from all modules?

Reply:
Actually this can be a one liner since we can use value.ToString() so we don't need this function

Reply:
yes, true